build-logic/src/main/kotlin/buildlogic.java-common-conventions.gradle.kts

```diff
@@ -1,11 +1,17 @@
 plugins {
     java
+    `jvm-test-suite`
```

Comment:
💄 You don't need to add `jvm-test-suite`. `java` adds it automatically already

Reply:
Ah, nice. I never heared of that plugin before; and I found myself being asked to integrate it, because of the link at https://github.com/gradlex-org/java-module-testing#whitebox-test-suites.

Reply:
It is a "newish" feature on top of Gradle's testing functionality that aims at providing a singe point for configuring test setup. For example to declare an additional test set:
https://docs.gradle.org/current/userguide/jvm_test_suite_plugin.html#sec:declare_an_additional_test_suite

(It does not add anything fundamentally new, but makes custom test configurations simpler - hence the module testing plugin builds on top of it)